### PR TITLE
Cl extend host calls

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

File: host_calls/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[package]
+name = "host_calls"
+version = "0.1.0"
+authors = ["clangenbacher <christian.langenbacher@scs.ch>"]
```
Comment (electronix) [inline host_calls/Cargo.toml:4]: Bitte richtigen Text einfügen

File: host_calls/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,300 @@
+#![feature(rustc_private)]
```
Comment (electronix) [inline host_calls/src/lib.rs:1]: Copyright header?

File: runtime/src/substratee_registry.rs [hunk 3]
```diff
@@ -0,0 +1,238 @@
+use rstd::prelude::*;
```
Comment (electronix) [inline runtime/src/substratee_registry.rs:1]: Copyright header

**Review by electronix — COMMENTED** [review]
See first comments. Not complete review done

File: host_calls/Cargo.toml (hunk 1 above)
Comment (clangenb) [reply host_calls/Cargo.toml:4]: done in master

File: host_calls/src/lib.rs (hunk 2 above)
Comment (clangenb) [reply host_calls/src/lib.rs:1]: done in master

File: runtime/src/substratee_registry.rs (hunk 3 above)
Comment (clangenb) [reply runtime/src/substratee_registry.rs:1]: done in master

**Pull request merged**